lib/ghtorrent/commands/repo_updater.rb

```diff
@@ -35,7 +35,7 @@ def set_deleted(owner, repo)
         where(Sequel.qualify('projects','owner_id') => Sequel.qualify('users','id')).\
         where(Sequel.qualify('users','login') => owner).\
         where(Sequel.qualify('projects','name') => repo).\
-        update(Sequel.qualify('projects','deleted') => true)
+        update(:deleted => true)
```

Comment:
If you are sure this work (I didn't test it yet), can you please apply it to the rest of the conditions for consistency?

Reply:
We learned that for the Update operation in PostgreSQL, this change was needed because PostgreSQL will not perform an update with the "table_name"."column" syntax. This will continue to work for MySQL users, but is needed for PostgreSQL.